repository/c.json

```diff
@@ -2204,6 +2191,18 @@
 				}
 			]
 		},
+		{
+			"name": "CodeLines",
+			"details": "https://github.com/absop/ST-CodeLines",
```

Comment:
You need to add a "previous_names" key for CodeCounter here so that existing users are moved over to the new name of the package. Look in the repository for examples.

Reply:
Thanks for your reminding, I have done that.